src/CMakeLists.txt

```diff
@@ -0,0 +1,15 @@
+set(TARGET piTest)
+set(SOURCES
+	piTest.c
+	piControlIf.c
+)
+
+add_executable(${TARGET} ${SOURCES})
+target_include_directories(${TARGET} PRIVATE ../inc ../lib/piControl)
+
+install(TARGETS ${TARGET} DESTINATION usr/bin)
```

Comment:
Please use [GNUInstallDirs](https://cmake.org/cmake/help/latest/module/GNUInstallDirs.html) for paths. See https://github.com/RevolutionPi/revpi-tools/blob/master/CMakeLists.txt 

I think `RUNTIME` needs to be added as well: https://cmake.org/cmake/help/latest/command/install.html#installing-targets